src/views/unauthenticated/About.tsx

```diff
@@ -0,0 +1,159 @@
+						If someone else needs to be let into to the shopping time you can
+						easily share specific lists with with in the list manager so they
+						can see and mark items as purchased too!
+					</li>
+				</ul>
+			</section>
+			<section>
+				<h2>Creators</h2>
+				<div>
+					<p>Maha Ahmed</p>
+					<Link to="https://www.linkedin.com/in/maha-ahmed3/" target="_blank">
+						<svg
+							xmlns="http://www.w3.org/2000/svg"
+							x="0px"
+							y="0px"
+							width="35"
+							height="35"
+							viewBox="0 0 50 50"
+						>
+							<path d="M 9 4 C 6.2504839 4 4 6.2504839 4 9 L 4 41 C 4 43.749516 6.2504839 46 9 46 L 41 46 C 43.749516 46 46 43.749516 46 41 L 46 9 C 46 6.2504839 43.749516 4 41 4 L 9 4 z M 9 6 L 41 6 C 42.668484 6 44 7.3315161 44 9 L 44 41 C 44 42.668484 42.668484 44 41 44 L 9 44 C 7.3315161 44 6 42.668484 6 41 L 6 9 C 6 7.3315161 7.3315161 6 9 6 z M 14 11.011719 C 12.904779 11.011719 11.919219 11.339079 11.189453 11.953125 C 10.459687 12.567171 10.011719 13.484511 10.011719 14.466797 C 10.011719 16.333977 11.631285 17.789609 13.691406 17.933594 A 0.98809878 0.98809878 0 0 0 13.695312 17.935547 A 0.98809878 0.98809878 0 0 0 14 17.988281 C 16.27301 17.988281 17.988281 16.396083 17.988281 14.466797 A 0.98809878 0.98809878 0 0 0 17.986328 14.414062 C 17.884577 12.513831 16.190443 11.011719 14 11.011719 z M 14 12.988281 C 15.392231 12.988281 15.94197 13.610038 16.001953 14.492188 C 15.989803 15.348434 15.460091 16.011719 14 16.011719 C 12.614594 16.011719 11.988281 15.302225 11.988281 14.466797 C 11.988281 14.049083 12.140703 13.734298 12.460938 13.464844 C 12.78117 13.19539 13.295221 12.988281 14 12.988281 z M 11 19 A 1.0001 1.0001 0 0 0 10 20 L 10 39 A 1.0001 1.0001 0 0 0 11 40 L 17 40 A 1.0001 1.0001 0 0 0 18 39 L 18 33.134766 L 18 20 A 1.0001 1.0001 0 0 0 17 19 L 11 19 z M 20 19 A 1.0001 1.0001 0 0 0 19 20 L 19 39 A 1.0001 1.0001 0 0 0 20 40 L 26 40 A 1.0001 1.0001 0 0 0 27 39 L 27 29 C 27 28.170333 27.226394 27.345035 27.625 26.804688 C 28.023606 26.264339 28.526466 25.940057 29.482422 25.957031 C 30.468166 25.973981 30.989999 26.311669 31.384766 26.841797 C 31.779532 27.371924 32 28.166667 32 29 L 32 39 A 1.0001 1.0001 0 0 0 33 40 L 39 40 A 1.0001 1.0001 0 0 0 40 39 L 40 28.261719 C 40 25.300181 39.122788 22.95433 37.619141 21.367188 C 36.115493 19.780044 34.024172 19 31.8125 19 C 29.710483 19 28.110853 19.704889 27 20.423828 L 27 20 A 1.0001 1.0001 0 0 0 26 19 L 20 19 z M 12 21 L 16 21 L 16 33.134766 L 16 38 L 12 38 L 12 21 z M 21 21 L 25 21 L 25 22.560547 A 1.0001 1.0001 0 0 0 26.798828 23.162109 C 26.798828 23.162109 28.369194 21 31.8125 21 C 33.565828 21 35.069366 21.582581 36.167969 22.742188 C 37.266572 23.901794 38 25.688257 38 28.261719 L 38 38 L 34 38 L 34 29 C 34 27.833333 33.720468 26.627107 32.990234 25.646484 C 32.260001 24.665862 31.031834 23.983076 29.517578 23.957031 C 27.995534 23.930001 26.747519 24.626988 26.015625 25.619141 C 25.283731 26.611293 25 27.829667 25 29 L 25 38 L 21 38 L 21 21 z"></path>
```

Comment:
I'd give [this article a read ](https://blog.logrocket.com/how-to-use-svgs-react/#:~:text=export%20default%20App%3B-,Adding%20SVG%20directly%20as%20JSX,be%20converted%20to%20JSX%20syntax.)if you haven't explored the different React-y ways of working with SVGs before 🙂 

One thing it suggests, which is my person preference, is to put svgs in their own file and import them into your component. I like this because svgs aren't all that readable, so moving them out of the way of code that needs to be read/reviewed is a really helpful way of keeping things organized.

I personally favor the solution under the header `Using the <img> tag for static SVGs` but if that isn't a quick win I really wouldn't suggest trying to get webpack configured with fileloader. That's a guaranteed way to hit a wall. If you really want to level up your frontend dev game I'd encourage reading up on webpack, but its generally a very senior-y thing to swim into that isn't for the light of heart (it's possibly my least favorite part of modern frontend web dev) 

_If_ you run into trouble there, the other way to do this could be to the option under the "Using an SVG as a component" header. You can declare them as their own component (with no logic) so you still get that separation of application logic and svg

Reply:
I will have to read up on this myself! For the SVG-as-a-component as a secondary option, that is what I had done  in the past, especially since I could manipulate the sizing of the SVG depending on the screen size of the user. The file structure looked liked this 
![image](https://github.com/user-attachments/assets/9fa3405b-c8f7-4cd0-bca7-e869c9491804). But I myself will look at the article Tanner linked as a refresher!